src/types.h

```diff
@@ -703,6 +703,16 @@ uint64_t asyncactionint(AsyncAction a);
 
 enum class PositionalParameterType { positional, count };
 
+namespace globalvars {
```

Comment:
nit: do we need this additional namespace?

Reply:
Well...all the other globalvar stuff is in bpftrace::globalvars namespace in globalvars.h so feels nice to be consistent. We only need to move the enum here to avoid a circular dependency between resources.h <-> globalvars.h

With that said, I don't feel strongly